x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/CompositeAggCursor.java

```diff
@@ -129,7 +134,7 @@ public void nextPage(SqlConfiguration cfg, Client client, NamedWriteableRegistry
             log.trace("About to execute composite query {} on {}", StringUtils.toString(query), indices);
         }
 
-        SearchRequest request = Querier.prepareRequest(query, cfg.requestTimeout(), includeFrozen, indices);
+        SearchRequest request = prepareRequest(query, includeFrozen, indices);
```

Comment:
where is the timeout configured now?

Reply:
I don't remember why but this must have fallen through the cracks during some refactoring. Will add it back (and consider adding a test).